packages/react/src/providers/FuelUIProvider.tsx

```diff
@@ -13,41 +13,45 @@ import { useConnect } from '../hooks/useConnect';
 import { useConnectors } from '../hooks/useConnectors';
 
 import { NATIVE_CONNECTORS } from '../config';
-import { useAccount, useBalance } from '../hooks';
-import { useFuel } from './FuelHooksProvider';
+import { useIsConnected } from '../hooks';
+import type { UIConfig } from '../types';
+import { isNativeConnector } from '../utils';
 
 export type FuelUIProviderProps = {
   children?: ReactNode;
+  uiConfig: UIConfig;
   fuelConfig: FuelConfig;
   theme?: string;
-  bridgeURL?: string;
 };
 
 export enum Routes {
+  LIST = 'list',
   INSTALL = 'install',
   CONNECTING = 'connecting',
-  BRIDGE = 'bridge',
   EXTERNAL_DISCLAIMER = 'disclaimer',
 }
 
 export type FuelUIContextType = {
-  bridgeURL?: string;
+  isConnected: boolean;
+  uiConfig: UIConfig;
   fuelConfig: FuelConfig;
   theme: string;
   connectors: Array<FuelConnector>;
   isLoading: boolean;
   isConnecting: boolean;
   isError: boolean;
   connect: () => void;
-  cancel: (ignoreBalance?: boolean) => void;
+  cancel: (params?: {
+    clean?: boolean;
+  }) => void;
   setError: (error: Error | null) => void;
   error: Error | null;
   dialog: {
     connector: FuelConnector | null;
     isOpen: boolean;
     back: () => void;
     connect: (connector: FuelConnector) => void;
-    retryConnect: () => Promise<void>;
+    retryConnect: (connector: FuelConnector) => Promise<void>;
```

Comment:
make it optional to avoid breaking change?

Reply:
This is new is not released yet.